vega_sim/scenario/common/agents.py

```diff
@@ -56,13 +56,12 @@ def initialise(self, vega: VegaServiceNull):
         ][0]
 
         # Get asset id
-
         asset_id = self.vega.find_asset_id(symbol=self.asset_name)
         # Top up asset
         self.vega.mint(
             self.wallet_name,
             asset=asset_id,
-            amount=100000,
+            amount=1e15,
```

Comment:
Why does this need to be so vast? 

This should also probably be a parameter when configuring the agent (called `initial_asset_mint` or something more exciting) rather than a hardcoded value, so if you could pull out that would be great (given I assume the issue here is that you need it 1e15 for the sim)